embedding_transfer_learning/run.py

```diff
@@ -1,61 +1,91 @@
-from art.checks import CheckResultExists, CheckScoreExists, CheckScoreCloseTo, CheckScoreLessThan, CheckScoreGreaterThan
-from art.steps import EvaluateBaseline, CheckLossOnInit, OverfitOneBatch, Overfit, Regularize
-from art.loggers import TensorBoardLoggerAdapter
+from dataset import EmbeddingDataModule
 from losses import ApproximateMRR
 from metrics import HitAtKMetric, MRRMetric
+from models.base_model import EmbeddingBaseline, EmbeddingHead
 from steps import DataPreparation, TextDataAnalysis
-from models.base_model import EmbeddingHead, EmbeddingBaseline
-from dataset import EmbeddingDataModule
 
+from art.checks import (
+    CheckResultExists,
+    CheckScoreCloseTo,
+    CheckScoreExists,
+    CheckScoreGreaterThan,
+    CheckScoreLessThan,
+)
+from art.loggers import TensorBoardLoggerAdapter
 from art.project import ArtProject
+from art.steps import (
+    CheckLossOnInit,
+    EvaluateBaseline,
+    Overfit,
+    OverfitOneBatch,
+    Regularize,
+)
 
 
 def getLogger(run_name: str):
     return TensorBoardLoggerAdapter(name=run_name)
 
+
 def main():
     project_name = "embedding_transfer_learning"
-    data_module = EmbeddingDataModule(batch_size=256)
+    data_module = EmbeddingDataModule(batch_size=256, num_workers=0)
```

Comment:
I now it needs to be on windows but it hurts

Reply:
Ill add a parser